lightning/src/ln/router.rs

```diff
@@ -1091,22 +1130,37 @@ mod tests {
 
 		let zero_hash = Sha256dHash::hash(&[0; 32]);
 
+		macro_rules! id_to_feature_flags {
```

Comment:
I don't understand the relationship between a channel(?) id and feature flags. Could you clarify?

Reply:
Added a comment, hopefully thats a bit more descriptive.